tools/local-storage/__tests__/storage.test.ts

```diff
@@ -0,0 +1,51 @@
+import storage from 'local-storage-fallback'
+import { get, set, getJson, setJson } from '../src/storage'
+
+beforeEach(() => {
+  storage.clear()
+})
+
+describe('get', () => {
+  it('returns a string keyed under "chainlink." in localStorage', () => {
+    storage.setItem('chainlink.foo', 'FOO')
+    expect(get('foo')).toEqual('FOO')
+  })
+
+  it('returns null when the key does not exist', () => {
+    expect(get('foo')).toEqual(null)
+  })
+})
+
+describe('getJson', () => {
+  it('returns a JS object for JSON keyed under "chainlink." in localStorage', () => {
+    storage.setItem('chainlink.foo', '{"foo":"FOO"}')
+    expect(getJson('foo')).toEqual({ foo: 'FOO' })
+  })
+
+  it('returns an empty JS object when not valid JSON', () => {
```

Comment:
This would be clearer as `returns an empty JS object when it retrieves invalid JSON from storage`.